### PR TITLE
EMX: Add CCPA Suport

### PR DESCRIPTION
**Pull request by ncolletti**
## Type of Change

- [ ] New adapter
- [x] Feature Update
- [ ] Bug Fix
- [ ] Code Refactor
- [ ] Other

## Description of Change
Add US CCPA Support to Adapter

### CODE REVIEW

**Comment by ncolletti** [comment]
FYI, we are not seeing the `usp` property of `ComplianceService` defined.

**Comment by jas-singh** [comment]
That is expected in the dev environment provided by the [Adapter Development Suite (ADS)](https://kb.indexexchange.com/Adapters/Adapter_Development_Suite.htm). 

In production, however, `usp` property will always be available if publisher has enabled IX library's privacy features (which can be determined by querying` ComplianceService.isPrivacyEnabled()` in the adapter). Additionally, the `uspString` property of the object retrieved from `ComplianceService.usp.getConsent()` will also exist as well. Please note the `uspString` property will be set as an empty string in the scenarios where the IX Library is not able to retrieve consent/opt-out from the page

File: emx-digital/emx-digital-htb.js [hunk 1]
```diff
@@ -169,6 +169,12 @@ function BRealTimeHtb(configs) {
                     }
                 };
             }
+
+            if (ccpaStatus) {
+                __emxData.us_privacy = ccpaStatus;
```
Comment (jas-singh) [inline emx-digital/emx-digital-htb.js]: Hi @ncolletti 

ccpaStatus in this case would be an object containing the following info provided by the CMP:
   - version: A number indicating the U.S. Privacy spec version
   - uspString: The U.S. Privacy string as defined in the IAB CCPA Compliance Framework for Publishers and Technology Companies 
e.g:
{
  version: 1,
  uspString: "1YNN"
}

Comment (ncolletti) [reply emx-digital/emx-digital-htb.js]: Thanks -- latest commit is now targeting 'uspString'

File: emx-digital/emx-digital-htb.js [hunk 2]
```diff
@@ -151,7 +152,7 @@ function BRealTimeHtb(configs) {
             }
         };
 
-        if (gdprPrivacyEnabled) {
+        if (privacyEnabled) {
             /* eslint-disable camelcase */
```
Comment (ix-certification) [inline emx-digital/emx-digital-htb.js]: Please remove all lint disables, and fix the linting issues ...

**Pull request merged**